### ISSUE TITLE
VERGE appears to ignore some VERGE.conf settings and hangs.

### ISSUE THREAD

**Issue by ShapeShifter499** [vergecurrency/VERGE#41]
VERGE on my end is compiled from source and it appears to be hanging, never loading the QT GUI. When I check the debug.log it seems like it's mining for coins although I have set gen=0.  Check the attached files.

[VERGE.conf](https://github.com/vergecurrency/VERGE/files/480062/VERGE.txt)
[debug.log](https://github.com/vergecurrency/VERGE/files/480061/debug.txt)

**Comment by vergecurrency** [comment]
on what os? arch?

**Comment by vergecurrency** [comment]
also which version of qt are you using? this requires qt webkit, so that might be the issue

**Comment by ShapeShifter499** [comment]
I have the qt5 set including webkit

**Comment by vergecurrency** [comment]
hmm. something had to have happened during compilation.. do you have a log of the configure or make still?

**Comment by vergecurrency** [comment]
also just to clarify, the daemon is working properly on arch, just not the qt wallet?

**Comment by ShapeShifter499** [comment]
Both VERGEd and VERGE-qt isn't working correctly for me.   Attached is a build log.
[build.txt](https://github.com/vergecurrency/VERGE/files/481693/build.txt)

**Comment by ShapeShifter499** [comment]
New log attached

VERGE-qt --debug
[debug.txt](https://github.com/vergecurrency/VERGE/files/481895/debug.txt)

**Comment by vergecurrency** [comment]
well i've narrowed it down to a lib or qt thats included in arch and the newest ubuntu (16.04). ill keep this issue open until i've got it patched. i was able to reproduce this result on ubuntu 16.04, and it's syncing even though it's still showing the Done Loading screen and the wallet hasn't opened. When it finishes, if it opens, it'll be much easier to narrow down. thanks for reporting this and supplying information to me =]

**Comment by justinvforvendetta** [comment]
@ShapeShifter499 is this still happening?

**Comment by Fluffkin** [comment]
I'm not ShapeShifter499, but yes it's still happening here. I suspect it's an "improvement" in QT. But I also get a hung wallet that doesn't get past the splash screen. It's busy downloading blocks in the background. I don't know yet if the gui appears when it's caught up.

**Comment by Fluffkin** [comment]
Nope ... the github compiled version still hangs in Linux land even after all the blocks are loaded.

**Comment by justinvforvendetta** [comment]
@Fluffkin i'll get to the bottom of this..

**Comment by buzztiaan** [comment]
bottom reached yet? :D

**Comment by Fluffkin** [comment]
Hehehe .. I fixed it .. but can't remember how .. (yes! one of those posts! sorry internet!)

**Comment by buzztiaan** [comment]
dangit internet!

**Comment by buzztiaan** [comment]
plz close this issue already @justinvforvendetta  :P

**Comment by justinvforvendetta** [comment]
never! @buzztiaan

**Comment by PrintedNOTmade** [comment]
This linux GUI hang issue was brought up like 16 months ago in v2.1 to the DEV by multiple people in IRC,  even a couple times since then check the channel logs ;) 
I am quite surprised it still has not been addressed till this day #Progress

**Comment by justinvforvendetta** [comment]
@PrintedNOTmade we stop pursuing the issue when @fluffkin said he fixed it, @Fluffkin what did you do to fix this on your end?

**Comment by Fluffkin** [comment]
Wakes up ... Eh?
Well .. it was one of those .. I compiled it for the Nth attempt .. wandered off, did other things for a few days, came back to it, ran the binary and it worked .. occasions ..

I *think* it was related to my Qt5 installation at the time. The current Qt5 configuration in Gentoo seems fine with producing a running version, and I'm fairly sure the Qt4 version (even though it looks like it compiles ok), doesn't play nice. I also seem to remember trying Qt5 earlier and it didn't work initially, so I have to assume I had something borked that then got upgraded.

Sorry folks, it's all I've got!
